### PR TITLE
Add documentation where missing for connectors.

### PR DESCRIPTION
**Pull request by aidapsibr**

### CODE REVIEW

File: src/REstate/IoC/RegistrarConnectorExtensions.cs [hunk 1]
```diff
@@ -4,22 +4,70 @@
...
+        /// </typeparam>
+        /// <param name="registrar"></param>
+        /// <param name="configuration">
+        /// A configuration object that includes an Identifier 
+        /// that can be resolved using a <see cref="Schematics.ConnectorKey"/> in a <see cref="Schematics.ISchematic{TState,TInput}"/>.
+        /// </param>
+        /// <param name="registrationName">
+        /// The name to register the type as in the <see cref="IComponentContainer"/>.
+        /// <para />
+        /// Multiple registrations with the same name will override each other.
```
Comment (abalfour) [inline src/REstate/IoC/RegistrarConnectorExtensions.cs]: Is this true?

File: src/REstate/Engine/Connectors/IConnectorConfiguration.cs [hunk 2]
```diff
@@ -3,12 +3,14 @@
...
-    /// A configuration object that can be associated with a connector.
+    /// A configuration object that can be associated with a <see cref="IConnector"/>.
     /// </summary>
     public interface IConnectorConfiguration
     {
         /// <summary>
-        /// A resolution string that can be used as a <see cref="ConnectorKey"/> in a Schematic.
+        /// A resolution string that can be used as a <see cref="ConnectorKey"/> in a <see cref="Schematic{TState,TInput}"/>.
+        /// <para />
+        /// Identifiers should ALWAYS be unique, overriding is NOT supported for safety/clarity reasons.
```
Comment (abalfour) [inline src/REstate/Engine/Connectors/IConnectorConfiguration.cs:13]: Is this true?

**Review by abalfour — DISMISSED** [review]

File: src/REstate/Engine/Connectors/IConnectorConfiguration.cs (hunk 2 above)
Comment (aidapsibr) [reply src/REstate/Engine/Connectors/IConnectorConfiguration.cs:13]: Yes, this is the true behavior.

File: src/REstate/IoC/RegistrarConnectorExtensions.cs (hunk 1 above)
Comment (aidapsibr) [reply src/REstate/IoC/RegistrarConnectorExtensions.cs]: Nope! updating.

**Pull request merged**